packages/snippet-bot/package.json

```diff
@@ -29,9 +29,9 @@
   },
   "dependencies": {
     "@google-cloud/storage": "^5.4.0",
-    "gcf-utils": "^6.2.0",
-    "node-fetch": "^2.6.0",
+    "gcf-utils": "^7.1.1",
     "minimatch": "^3.0.4",
+    "node-fetch": "^2.6.0",
```

Comment:
Do you end up using this?

Reply:
Thanks, although I think it's a good idea, we should tackle that in another PR in my opinion. WDYT?